article/_posts/2019-07-17-HDML.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: review
+title: "Hardness-Aware Deep Metric Learning"
+tags: deep-learning autoencoder
+author: "Pierre-Marc Jodoin"
+cite:
+    authors: "Wenzhao Zheng, Zhaodong Chen, Jiwen Lu, and Jie Zhou"
+    title:   "Hardness-Aware Deep Metric Learning"
+    venue:   "CVPR 2019"
+pdf: "https://arxiv.org/pdf/1903.05503.pdf"
+---
+
+
+# Introduction
+The goal of this paper is to learn deep metrics.  A deep metric is a function which computes the similarity between two input signals (here images).  This may be done with a contrastive loss or a triplet loss which both consider a set of 3 (triplet loss) or 4 (contrastive loss) positive and negative (read similar and different) images to teach a neural net to project those points into a low-dimensional Euclidean space in which similar pointis are close and dissimilar points are far. 
+
+The authors argue that the more a neural net is shown difficiult sets of points (negatives images with similar content or positive images with different content) the better that system will be.  In that perspective, the author proposed a novel **hardness-ware strategy**.  This paper sits on three main ideas namely:
+
+* From a negative pair of points, generate a new and yet more difficult negative point (c.f. Fig.1 - going from $$y^-$$ to $$\hat{y}^-$$ and from $$z^-$$ to $$\hat{z}^-$$)
+* Make sure that the newly generated negative points still correspond to a viable image (c.f. Fig.1 - going from $$\hat{y}^-$$ to $$\tilde{y}^-$$ and from $$\hat{z}^-$$ to $$\tilde{z}^-$$)
+* Make the difficulty of the synthetic negative points increase during training (c.f. Fig.2).
+
+![](/article/images/hdml/sc01.jpg)
+![](/article/images/hdml/sc02.jpg)
+
+
+# Methods
+
+The overall system is illustrated in Fig.3.
+![](/article/images/hdml/sc03.jpg)
+
+It starts with a GoogleNet which projects the input image to a vector space $$Y$$.  The follows an autoencoder which learns a Euclidean latent space $$Z$$.  The synthetic "harder" example is then generated as follows
```

Comment:
_The follows an autoencoder (...)_ -> _Then an autoencoder follows (...)_? or _It is followed by an autoencoder (...)_ ?

Reply:
done